packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -512,8 +514,32 @@ export default {
 
 .c-cookieBanner-content {
     margin: 0 auto;
-    padding: f.spacing(f);
+    padding: 0 f.spacing(d);
     text-align: left;
+    overflow-y: auto;
+
+    @include f.media('>=mid') {
+        padding: 0 f.spacing(f);
+    }
+
+    @include f.media('<mid') {
+        background:
+            // Scroll shadow cover (hides the shadow when you scroll down)
+            linear-gradient(
+                rgba(255,255,255,0),
+                white 70%
+            ) center bottom,
+
+            // Scroll shadow
+            radial-gradient(
+            farthest-side at 50% 100%,
+                rgba(0,0,0,0.3),
+                rgba(0,0,0,0)
+            ) center bottom;
+        background-repeat: no-repeat;
+        background-size: 100% 48px, 1000% 12px;
```

Comment:
can we reference `f.spacing`? for 48px we might need to add some values together

Reply:
We _could_, but I'm not sure what it would achieve to be honest. If we changed the value of the underlying spacing variables then I'm not sure we would want it to affect these styles because it has little to do with the spacings between elements.